tests/e2e/utils.go

```diff
@@ -51,17 +59,18 @@ func NewFunnel() *Funnel {
 	conf = testutils.TempDirConfig(conf)
 	conf = testutils.RandomPortConfig(conf)
 	conf.Logger = logger.DebugConfig()
-	conf.Worker.LogUpdateRate = rate
-	conf.Worker.UpdateRate = rate
+	conf.Worker.LogUpdateRate = time.Millisecond * 1300
+	conf.Worker.UpdateRate = time.Millisecond * 1300
 	conf.Worker.Logger = logger.DebugConfig()
-	conf.ScheduleRate = rate
+	conf.ScheduleRate = time.Millisecond * 700
+	conf.Worker = config.WorkerInheritConfigVals(conf)
 
 	storageDir, _ := ioutil.TempDir("./test_tmp", "funnel-test-storage-")
 	wd, _ := os.Getwd()
 
 	conf.Storage = config.StorageConfig{
 		Local: config.LocalStorage{
-			AllowedDirs: []string{storageDir, wd},
+			AllowedDirs: []string{"./test_tmp", wd},
```

Comment:
Is this necessary? Does the server need access to storage outside of the storageDir?